### DISCUSSION TITLE
Not possible to have multiple sessions

### DISCUSSION THREAD

**Discussion by awallemo** [cypress-io/cypress#25222] [Questions and Help]
I want to have multiple sessions enabled for a test, but it is only the last session that becomes active, all others are cleared (or the cookies within these sessions are cleared). Am I misunderstanding the concept here? Does anyone have suggestions to what I could do?   

` beforeEach(() => {
            cy.loginMock()
            cy.setReportEntitiesFilter(Cypress.env("DPR_ENTITY_CODE"))
            cy.setDeliveryStatesFilter('Success')
            cy.setReportTypesFilter('DPR')
        });`

All of the above functions are each their session, but for each new session, the others are cleared and I can't do what I want to.

**Comment by emilyrohrbough** [comment]
@awallemo At this time, we don't support multiple sessions at the same time. I created a feature request from this discussion (https://github.com/cypress-io/cypress/issues/25227). Can you explain on the issue why you'd like to apply multiple sessions in one test vs creating a single session with the session data expected? Explaining your needs will help use evaluate this request and what needs to be considered if we decide to implemented it.
